test_app/tests/rbac/api/test_rbac_views.py

```diff
@@ -74,7 +74,11 @@ def test_make_user_assignment(admin_api_client, inv_rd, rando, inventory):
     data = dict(role_definition=inv_rd.id, user=rando.id, object_id=inventory.id)
     response = admin_api_client.post(url, data=data, format="json")
     assert response.status_code == 201, response.data
-    assert response.data['created_by']
```

Comment:
With this change, read-only fields like `created_by` will not be returned in the POST response data. This is why we want to be selective about enabling this.